README.md

```diff
@@ -100,6 +100,8 @@ Important notice is that it is assumed that the http client is a transient or sc
 
 ## Coupling with a Service Agent
 
+### > v5.0.0
```

Comment:
Hier lees ik "groter dan v5.0.0" maar dat klopt niet denk ik. Vervangen door "< v5.0.0" of misschien duidelijker "Until v4".